Makefile

```diff
@@ -37,7 +37,7 @@ vendor:
 	go mod verify
 
 check-vendor: vendor
-	git diff --quiet HEAD -- go.mod go.sum vendor
+	git diff --exit-code HEAD -- go.mod go.sum vendor
```

Comment:
for the curious reader: exits with non-zero code if there are differences

(was this a bug?)